spacy/lang/sl/tokenizer_exceptions.py

```diff
@@ -0,0 +1,260 @@
+from typing import Dict, List
+from ..tokenizer_exceptions import BASE_EXCEPTIONS
+from ...symbols import ORTH, NORM
+from ...util import update_exc
+
+_exc: Dict[str, List[Dict]] = {}
+
+_other_exc = {
+    "...": [{ORTH: "."}, {ORTH: "."}, {ORTH: "."}],
```

Comment:
Is there a particular reason why we would want to split `...` into three tokens for Slovenian? We normally regard it as a single token with its own semantics: the fact it contains three periods could be seen as an implementation detail.

Reply:
There is not, I just followed our gold standard dataset that seems to split it. But if Spacy in general does not split it, I think it also makes sense to be that way for Slovenian. I removed the exception.